cmd/check_container.go

```diff
@@ -226,6 +228,19 @@ var checkContainerCmd = &cobra.Command{
 	},
 }
 
+func pyxisHostLookup(pyxisEnv, hostOverride string) string {
+	envs := map[string]string{
+		"prod": "catalog.redhat.com/api/containers",
+		"uat":  "catalog.uat.redhat.com/api/containers",
+		"qa":   "catalog.qa.redhat.com/api/containers",
+	}
+	if hostOverride != "" {
+		return hostOverride
+	}
+
+	return envs[pyxisEnv]
+}
```

Comment:
I think `pyxisEnv` needs validation here, right? 

If the user providers an invalid value for pyxisEnv, I think we panic on the return because the key wouldn't exist in our map.

Reply:
You're right. I wasn't sure what to do with that case. I guess just return default?